lib/manifest-generator.js

```diff
@@ -43,7 +43,8 @@ var ManifestGenerator = Addon.extend({
 
     var manifestOptions = objectAssign({
       appName: app.name,
-      generateURI: assetLoaderOptions.generateURI
+      generateURI: assetLoaderOptions.generateURI,
+      rootURL: assetLoaderOptions.rootURL || ''
```

Comment:
The `rootURL` value should likely come from the application's `config`, not the asset loader's options. Otherwise consumers will need to define it in two places.